src/main/java/com/lms/learning_management_system/entities/CourseEntity.java

```diff
@@ -0,0 +1,74 @@
+package com.lms.learning_management_system.entities;
+
+import jakarta.persistence.*;
+
+import java.util.List;
+import java.util.UUID;
+
```

Comment:
You should use lombok to decrease you code between line 23 and 73, if you need help with this message me